control/controlclient/direct.go

```diff
@@ -772,6 +772,9 @@ func (c *Direct) sendMapRequest(ctx context.Context, maxPolls int, cb func(*netm
 			go answerPing(c.logf, c.httpc, pr)
 		}
 
+		if !resp.ControlTime.IsZero() {
+			c.logf("netmap: control time is %v", resp.ControlTime.Format(time.RFC3339Nano))
```

Comment:
Convert `ControlTime` to UTC for consistent formatting?

Reply:
Ugh. Catches me every time. Thank you.